extensions/sql-migration/src/api/utils.ts

```diff
@@ -1190,6 +1193,74 @@ export async function refreshAuthenticationKeyTable(view: ModelView, table: Decl
 	});
 }
 
+//Function that creates stage status table.
+export function createIntegrationRuntimeTable(view: ModelView): DeclarativeTableComponent {
+	const rowCssStyle: CssStyles = {
+		'border': 'none',
+		'text-align': 'left',
+		'box-shadow': 'inset 0px -1px 0px #F3F2F1',
+		'font-size': '13px',
+		'line-height': '18px',
+		'padding': '7px 0px',
+		'margin': '0px',
+	};
+
+	const headerCssStyles: CssStyles = {
+		'border': 'none',
+		'text-align': 'left',
+		'box-shadow': 'inset 0px -1px 0px #F3F2F1',
+		'font-weight': 'bold',
+		'padding-left': '0px',
+		'padding-right': '0px',
+		'font-size': '13px',
+		'line-height': '18px'
+	};
+	const _integrationRuntimeTable: DeclarativeTableComponent = view.modelBuilder.declarativeTable().withProps({
+		columns: [
+			{
+				displayName: 'Node Name',
```

Comment:
all of these column label strings need to be added as resource strings in strings.ts for localization.